config/rbac/role.yaml

```diff
@@ -6,6 +6,17 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
```

Comment:
Why do we need two files (role.yaml and topolvm_controller_role.yaml) for defining roles/cluster roles?

Reply:
in rbac folder whatever isn't prefixed with topolvm corresponds to lvm operator rbac

For now, components will add their rbac and towards the end (after all units implementation) if anything is sharing the same roles those can be clubbed, operator rbac be renamed to correctly reflect their usage.